ts/screens/wallet/AddPaymentMethodScreen.tsx

```diff
@@ -72,68 +58,99 @@ const bancomat: IPaymentMethod = {
  * Keep in mind that the rest of the "add credit card" process
  * is handled @https://www.pivotaltracker.com/story/show/157838293
  */
-class AddPaymentMethodScreen extends React.PureComponent<Props> {
-  public render(): React.ReactNode {
-    const inPayment = this.props.navigation.getParam("inPayment");
+const AddPaymentMethodScreen: React.FunctionComponent<Props> = (
+  props: Props
+) => {
+  const inPayment = props.navigation.getParam("inPayment");
 
-    return (
-      <BaseScreenComponent
-        goBack={true}
-        contextualHelpMarkdown={contextualHelpMarkdown}
-        faqCategories={["wallet", "wallet_methods"]}
-        headerTitle={
-          inPayment.isSome()
-            ? I18n.t("wallet.payWith.header")
-            : I18n.t("wallet.addPaymentMethodTitle")
-        }
-      >
+  const cancelButtonProps = {
+    block: true,
+    light: true,
+    bordered: true,
+    onPress: props.navigateBack,
+    title: inPayment.isSome()
+      ? I18n.t("global.buttons.back")
+      : I18n.t("global.buttons.cancel")
+  };
+
+  const paymentMethods: ReadonlyArray<IPaymentMethod> = [
```

Comment:
this could be placed outside the body of the component?

Reply:
I thought the same, but it may cause the common issue that ignores the translation with locale selected by user

what do you think?